viewer/packages/tools/src/Measurement/MeasurementTool.ts

```diff
@@ -374,12 +377,16 @@ export class MeasurementTool extends Cognite3DViewerToolBase {
       window.addEventListener('keydown', this._handleMeasurementCancel);
     } else {
       this._activeMeasurement.endMeasurement(intersection.point);
+      const measurement = this._activeMeasurement.getMeasurement();
+
       this._measurements.push(this._activeMeasurement);
       this._activeMeasurement = undefined;
       // To avoid issue when exiting measurement mode when a measurement 'added' event called
       this._events.measurementAdded.fire(this._measurements[this._measurements.length - 1].getMeasurement());
       this._viewer.domElement.removeEventListener('pointermove', this._handlePointerMove);
       window.removeEventListener('keydown', this._handleMeasurementCancel);
+
+      MetricsLogger.trackEvent('measurementAdded', { measurement });
```

Comment:
Is the measurement really interesting for us to log? I'd think just the use itself, or maybe the current number of measurements in the model, would be nice.

Reply:
My thinking is it would provide us with an idea of what scale the tool is used for (e.g centimeters or kilometers)

Reply:
Alright, I'm not gonna fight you on this :P I guess some information like that could be useful